**isomdurm**/README.md

```diff
@@ -0,0 +1 @@
+#**isomdurm**'s Homework repo
```

Comment:
We're going to need to fix this @isomdurm  
I think there was a misunderstanding with the instructions.  You shouldn't name files or directories using `*` or `?` it'll cause you a lot of pain in the future since those characters are used for other things.  
`ls *.jpg` lists your jpg files.  

Could you rename this to `isomdurm` or just re-create it?  And then move all your other files and all future homeworks into that directory as well.